### PR TITLE
do not fil fossa build when running from a fork where FOSSA_API_KEY i…

### PR DESCRIPTION
**Pull request by grosser**
…s not available

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,5 +33,8 @@ matrix:
       before_script:
         - export BINDIR=$(pwd) && curl -H 'Cache-Control:no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | bash
       script:
+        - ls sddfssdfsdf # test
```
Comment (ragurney) [inline .travis.yml]: Remove this?

File: .travis.yml [hunk 2]
```diff
@@ -33,5 +33,8 @@ matrix:
...
+        - [[ -z "$FOSSA_API_KEY" ]] && [[ "$TRAVIS_PULL_REQUEST" != "false" ]] && travis_terminate 0 # pass on PRs from forks that do not get FOSSA_API_KEY
+        - echo test && travis_terminate 0 # test
```
Comment (ragurney) [inline .travis.yml]: and this?

Comment (grosser) [reply .travis.yml]: yeah, still testing if it works correctly :)

**Comment by grosser** [comment]
much better now :)

**Pull request merged**